docs/reference/method/MongoDBGridFSBucket-getFileIdForStream.txt

```diff
@@ -38,7 +38,27 @@ Errors/Exceptions
 .. include:: /includes/extracts/error-invalidargumentexception.rst
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = $bucket->openUploadStream('filename');
+
+   $id = $bucket->getFileIdForStream($stream);
+
+   var_dump($id);
```

Comment:
Same errors as https://github.com/mongodb/mongo-php-library/pull/527/files#r182849516.

Reply:
Add `fclose($stream)` after the `var_dump()` for now.